templates/lookup/ferm__combined_rules.j2

```diff
@@ -0,0 +1,63 @@
+{% macro merge_dict(current_dict, to_merge_dict, dict_key='name') %}
```

Comment:
Sure. Just remember that this code was written before the merge_dict macro existed... ;-)